docs/plugins.md

```diff
@@ -609,6 +609,11 @@ function getNextNonSpaceNonCommentCharacterIndex(
 function isNextLineEmpty(text: string, startIndex: number): boolean;
 
 function isPreviousLineEmpty(text: string, startIndex: number): boolean;
+
+function getPreferredQuote(
+  rawContent: string,
```

Comment:
Let's rename the first parameter to `text`, this function doesn't care if it's raw or cooked.